src/main/java/data/Database.java

```diff
@@ -0,0 +1,74 @@
+
+package data;
+
+import java.io.InputStream;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.ArrayList;
+
+/* This class creates an instance of a database that a file can be written to and read from.
+Assumptions: for each runID, only 1 of each datatype can be associated with it */
+public class Database{
+ 
+    /* Place where files are stored.*/
+    private String name;
+
+    /* List of all files stored in Database.*/
+    ArrayList<String> files;
+ 
+    /* Creates instance of a database with this name. */
+    public Database(String name){
+      /* Database will be GCS and buckets. Bucket will named by different runids. */
```

Comment:
The current database is the FileDatabase, if you want to add more other kinds of db, you can define new class e.g. GCSDatabase, then consider some abstract design.

Reply:
Will this be ok to do for the iteration after the MVP ?